ini/dsaber.ini

```diff
@@ -1,50 +1,49 @@
-[data]
-name     = Dragon Saber
-driver   = System2x
-type     = System2
-filename = snd0.bin
-filename = snd1.bin
-gain     = 1.0
-muterear = 0
-
-[wave.0]
-filename = voi1.bin
-length   = 0x80000
-position = 0x100000
-
-[wave.1]
-filename = voi2.bin
-length   = 0x80000
-position = 0x300000
-
-[playlist]
-108 = Story of Dragon Saber
-105 = Coin
-109 = Opening Visual
-121 = Submerged City
-111 = Boss A
-122 = Volcano
-112 = Boss B
-123 = Fossil
-113 = Alien King / Woody
-124 = Fungal Land
-125 = Canyon
-10d = Continue
-101 = Name Ranking A
-126 = Ice Cave
-127 = Devildom
-117 = Dragon Zombie
-128 = Dark
-129 = Ultima
-119 = Chaos
-10b = Ending
-102 = Name Ranking B
-005 = Dragon Spirit II - Coin
-021 = Dragon Spirit II - Round 1
-022 = Dragon Spirit II - Round 2
-023 = Dragon Spirit II - Round 3
-024 = Dragon Spirit II - Round 4
-025 = Dragon Spirit II - Round 5
-026 = Dragon Spirit II - Round 6
-00b = Dragon Spirit II - Ending
-003 = Dragon Spirit II - Name Ranking
+[data]
+name     = Dragon Saber
+driver   = System2x
+type     = System2
+filename = snd0.bin
+filename = snd1.bin
+gain     = 1.0
+muterear = 0
+
+[wave.0]
+filename = voi1.bin
+length   = 0x80000
+position = 0x100000
+
+[wave.1]
+filename = voi2.bin
+length   = 0x80000
+position = 0x300000
+
+[playlist]
+108 = Story of Dragon Saber
+105 = Credit Sound
+109 = Opening Visual
+121 = Submerged City
+111 = Boss A
+122 = Volcano
+112 = Boss B
+123 = Fossil
+113 = Alien King / Woody
+124 = Waste Land
+125 = Canyon
+10d = Continue
+101 = Name Ranking A
+126 = Ice Cave
+127 = Devildom
+117 = Dragon Zombie
+128 = Dark
+129 = Ultima
+119 = Chaos
+10b = Ending
+102 = Name Ranking B
+021 = Stage 1: Dragon Spirit System2 Arrange Version

```

Comment:
I'd rather keep the song titles from [this](https://vgmdb.net/album/560)

Reply:
It's up to you obviously, but I'd argue the other tracklisting is more accurate since it was used on both of these:
https://vgmdb.net/album/19429
https://vgmdb.net/album/42011